docs/teams.md

```diff
@@ -0,0 +1,30 @@
+# Teams BETA
+
+A team in Quilt is a separate package namespace and access control specification. Being part of a team lets you install and push team-visible packages.
```

Comment:
suggest "private package namespace" to emphasize security.

"Being part of a team" should be "As a team member" (so we use standard terminology of teams, members, etc.).